pkg/kf/apps/kfapp.go

```diff
@@ -152,6 +153,37 @@ func (k *KfApp) DeleteEnvVars(names []string) {
 	k.SetEnvVars(envutil.RemoveEnvVars(names, k.GetEnvVars()))
 }
 
+// SetMemory sets memory request for the app.
+func (k *KfApp) SetMemory(memory *resource.Quantity) {
+	k.setResourceRequest(corev1.ResourceMemory, memory)
+}
+
+// SetStorage sets disk storage request for the app.
```

Comment:
You should print out the newly supported resources in `pkg/kf/commands/apps/app.go`

Reply:
done, added to describe